src/main/java/cc/hyperium/mods/playtime/PlayTime.java

```diff
@@ -29,6 +28,10 @@ public Metadata getModMetadata() {
         return new Metadata(this, "Play Time", "1.0", "SHARDCoder");
     }
 
-    static long startSysTime;
-
+    @InvokeEvent
+    public void tickEvent(TickEvent event) {
+        Settings.TOTAL_PLAYTIME = startConfigTime + (System.currentTimeMillis() - PlayTime.startSysTime);
+        sessionPlayTime = (System.currentTimeMillis() - PlayTime.startSysTime) / 1000;
+        totalPlayTime = Settings.TOTAL_PLAYTIME / 1000;
+    }
```

Comment:
This can be calculated when the player runs the command

Reply:
this is for the total time value